### PR TITLE
docs(tests): end-to-end vs unit

### PR DESCRIPTION
**Pull request by alisonailea**
## Summary

Documentation only update to provide context to developers on writing end-to-end vs unit tests for their code.

### CODE REVIEW

File: conventions/Testing.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ We encourage writing expressive test cases and code that indicates intent. Use c
 
 Please see Stencil's doc for more info on [end-to-end](https://stenciljs.com/docs/end-to-end-testing) testing. See one of our test examples [here](https://github.com/Esri/calcite-components/blob/master/src/components/block/block.e2e.ts).
 
+## Should write end-to-end or unit tests?
```
Comment (jcfranco) [inline conventions/Testing.md:9]: Suggestion: `unit (spec) tests`

**Review by driskull — APPROVED** [review]
👍 Looks good @alisonailea

**Review by jcfranco — APPROVED** [review]
Aside from one comment regarding 3rd-party module testing, this LGTM! ✨📚✨

**Comment by jcfranco** [comment]
@benelan Not sure why the Screener check is holding this up since it's only markdown changes. Merging!

**Comment by benelan** [comment]
> Not sure why the Screener check is holding this up since it's only markdown changes. Merging!

Yeah the conditional I frantically added to skip running screener on PR syncs also skips the markdown check that happens in the same step. I'll fix that

**Comment by benelan** [comment]
FWIW adding the "pr is ready for visual snapshots" label would have passed the check since it is only markdown. But I can improve that

**Pull request merged**